rfc6962-bis.xml

```diff
@@ -1314,6 +1314,20 @@ but it is expected there will be a variety.
             Since each received SCT may be valid either for the server certificate itself or for a <xref target="name_constrained">suitable name-constrained intermediate</xref>, TLS clients MAY need to use trial and error to determine which certificate in the chain is the certificate to which a received SCT corresponds.
           </t>
         </section>
+        <section title="Reconstructing the TBSCertificate" anchor="reconstructing_tbscertificate">
```

Comment:
Moved the section - not entirely sure how you want to mention the reconstructed TBSCertificate in the  Validating SCTs section.